docs/markdown/android/icon.md

```diff
@@ -94,6 +86,48 @@ Used sparingly to draw attention to an icon that might otherwise be missed.
 </Group>
 </TwoCol>
 
+### Color
+Icon colors are semantic—they have a specific meaning and aren't arbitrary. There is no disabled color for icons, as that is handled by the button state that an icon is in.
+
+#### Light mode
+
+<iframe style={{border:0}} width="100%" height="490" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FREw1COFYAktmVWrUBh3Ov8%2FGestalt-for-Android%3Fnode-id%3D5058%253A18866%26t%3D0eckWD5MZ5KL7Upr-1" allowfullscreen></iframe>
+
+#### Dark mode
+<iframe style={{border:0}} width="100%" height="490" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FREw1COFYAktmVWrUBh3Ov8%2FGestalt-for-Android%3Fnode-id%3D5058%253A19057%26t%3DEOtvT24P2BprSIEh-1" allowfullscreen></iframe>
+
+
+### Platform-specific icons
+
+1. **Share**
+On Android, use the android-share icon to indicate the ability to share an element. 
+2. **Check**
+On Android, use the standard radio button instead of an icon.
+3. **Back**
+On Android use the arrow icon titled directional-arrow-left to indicate backward movement. 
+
+<ThreeCol>
+<Group>
+<ImgContainer src="https://i.pinimg.com/originals/ae/c6/de/aec6de6e9f1c871a22f4f0ff3afc8707.jpg" alt="Android's share icon." />
+android-share
+</Group>
+<Group>
+<ImgContainer src="https://i.pinimg.com/originals/fd/31/ca/fd31ca3526df5d7326a6cf0dfaee5edc.jpg" alt="A radio button instead of a check icon." />
+Radio Button (not an icon)
+</Group>
+<Group>
+<ImgContainer src="https://i.pinimg.com/originals/e8/a4/1c/e8a41cc5c6ce014dc50bfc801ca7fb01.jpg" alt="A arrow pointing left." />
+directional-arrow-left
+</Group>
+</ThreeCol>
+
+## Naming
+
+* Follow the web icon naming convention when possible
```

Comment:
icon naming convention< can we add a link to this conventions?

Reply:
good idea! added! since this is markdown, I can't currently include our "internal only icon".